package.json

```diff
@@ -54,7 +53,6 @@
     "@typescript-eslint/parser": "6.7.2",
     "@vitest/coverage-v8": "^1.2.1",
     "@vitest/browser": "^1.2.1",
-    "codecov": "^3.8.3",
```

Comment:
This package is deprecated. https://about.codecov.io/blog/codecov-uploader-deprecation-plan/